src/vinyldns/client.py

```diff
@@ -229,9 +268,11 @@ def list_my_groups(self, group_name_filter=None, start_from=None, max_items=None
             args.append(u'maxItems={0}'.format(max_items))
 
         url = urljoin(self.index_url, u'/groups') + u'?' + u'&'.join(args)
-        response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
+        status_code, data = self.__make_request(url, u'GET', self.headers, **kwargs)
```

Comment:
looks like this is just not wired in the right way.

iirc, `def __make_request` needs to be updated to call the `__check_response`, then no other methods need to be changed.

Reply:
Oops, didn't mean to include this.